.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Avoid checking-in editor specific settings. When someone pulls your code, their settings and this settings would conflict. Use `.gitignore`